producer.go

```diff
@@ -168,10 +189,15 @@ func (h *AsyncApexLogNSQHandler) HandleLog(e *log.Entry) error {
 
 	select {
 	case h.logChan <- e:
+		if h.logChanFull {
+			h.logChanFull = false
+		}
 		// Do nothing more
 	default:
-		backupLogger.Error("AsyncApexLogNSQHandler log channel is full")
-		backupLogger.Handler.HandleLog(e)
+		if !h.logChanFull {
+			backupLogger.Error("AsyncApexLogNSQHandler log channel is full")
```

Comment:
we don't want to suppress this log line, i would keep complaining about the log channel being full